interfaces/policy/helpers.go

```diff
@@ -31,24 +31,12 @@ import (
 
 // check helpers
 
-func snapIDSnapd(snapID string) bool {
-	var snapIDsSnapd = []string{
-		// production
-		"PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4",
-		// TODO: when snapd snap is uploaded to staging, replace this with
-		// the real snap-id.
-		"todo-staging-snapd-id",
-	}
-	return strutil.ListContains(snapIDsSnapd, snapID)
-}
-
-func checkSnapType(snap *snap.Info, types []string) error {
+func checkSnapType(snapInfo *snap.Info, types []string) error {
```

Comment:
TBH I'd be completely bamboozled when doing `checkSnapType(snapdSnapInfo, []string{"snapd"})` told me it didn't match